hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmSnapshotManager.java

```diff
@@ -268,4 +280,29 @@ private void verifySnapshotInfoForSnapDiff(final SnapshotInfo fromSnapshot,
           " should be older than to toSnapshot:" + toSnapshot.getName());
     }
   }
+
+  private RocksDB createRocksDbForSnapshotDiff(OzoneConfiguration config) {
```

Comment:
It seems like this new rocksdb should not be included when bootstraping a follower, is that right?

Reply:
Yes, for now. We may revisit this when we work on making snapdiff HA or guarantee the persistence of diff report.